src/libs/rfc-merge-procedure.md

```diff
@@ -0,0 +1,3 @@
+# RFC Merge Procedure
+
+The Libs team follows the [same merge procedure as the Lang team](../../lang/rfc-merge-procedure).
```

Comment:
```suggestion
The Libs team follows the [same merge procedure as the Lang team](../../lang/rfc-merge-procedure.html).
```

Reply:
Ah so we do want the `.html` extension on there? I'd just omitted it because the link worked without it so it seemed unnecessary.

Reply:
Hmm, it shouldn’t work. 😅 Are you testing it through `mdbook serve` because that probably explains why it works. I’ll have to double check but I’m pretty sure this will fail when on GitHub Pages or if you clicked this link with the page opened as a file in the browser.